pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -214,9 +216,11 @@ func (o *capability) Execute(ctx context.Context, r capabilities.CapabilityReque
 func (o *capability) queueRequestForProcessing(
 	ctx context.Context,
 	metadata capabilities.RequestMetadata,
-	i *inputs) (<-chan capabilities.CapabilityResponse, error) {
+	i *inputs,
+) (<-chan capabilities.CapabilityResponse, error) {
 	callbackCh := make(chan capabilities.CapabilityResponse, o.callbackChannelBufferSize)
 	r := &request{
+		// TODO: link this to the passed-in context
```

Comment:
This TODO got removed by someone else at some point, so putting it back to document it

Reply:
It was refactored deliberately, and the context was being passed through before this change.

Reply:
I'm not objecting to the refactoring, I'm objecting to the removal of the TODO: StopCh is analogous to context.Background() in terms of how it's used, but the underlying TODO (which is a broader issue around how the caller of a capability can cancel the ongoing request) hasn't been addressed.

Reply:
I've updated the comment to provide more context on the issue